emmaa_service/api.py

```diff
@@ -64,15 +63,16 @@ def _translator(status):
 
 def _get_model_meta_data():
     s3 = boto3.client('s3')
-    resp = s3.list_objects(Bucket=EMMAA_BUCKET_NAME, Prefix='models/',
+    resp = s3.list_objects(Bucket='emmaa', Prefix='models/',
```

Comment:
Not that it would make a huge difference but I guess we might as well import EMMAA_BUCKET_NAME from emmaa.model and use the variable here.

Reply:
I intentionally left it 'emmaa' in some places and a variable in others to make it easier to switch for local testing. We can switch EMMAA_BUCKET_NAME to 'emmaa_test' to write and read test results/stats from test bucket when making changes, but still want to get a list of all models and their configs from main bucket.